tests/components/elkm1/test_logbook.py

```diff
@@ -0,0 +1,65 @@
+"""The tests for elkm1 logbook."""
+from homeassistant.components.elkm1.const import (
+    ATTR_KEY,
+    ATTR_KEY_NAME,
+    ATTR_KEYPAD_ID,
+    ATTR_KEYPAD_NAME,
+    DOMAIN,
+    EVENT_ELKM1_KEYPAD_KEY_PRESSED,
+)
+from homeassistant.const import CONF_HOST
+from homeassistant.setup import async_setup_component
+
+from . import _patch_discovery, _patch_elk
+
+from tests.common import MockConfigEntry
+from tests.components.logbook.common import MockRow, mock_humanify
+
+
+async def test_humanify_elkm1_keypad_event(hass):
+    """Test humanifying elkm1 keypad presses."""
+    hass.config.components.add("recorder")
+    assert await async_setup_component(hass, "logbook", {})
+    config_entry = MockConfigEntry(
+        domain=DOMAIN,
+        data={CONF_HOST: "elks://1.2.3.4"},
+        unique_id="aa:bb:cc:dd:ee:ff",
+    )
+    config_entry.add_to_hass(hass)
+
+    with _patch_discovery(), _patch_elk():
+        await hass.config_entries.async_setup(config_entry.entry_id)
+        await hass.async_block_till_done()
+
+    await hass.async_block_till_done()
```

Comment:
Do we need an extra block till done?

Reply:
Nope ca218c4ebcb917e09f02d1b73ada3928711703da